documentation/workflows.md

```diff
@@ -0,0 +1,118 @@
+# Working on the Project
+
+Better to be explicit/specific on topics than to wait for avoidable confusion due to ambiguity.
+
+## Principles
+
+1. Changes are NEVER done directly on `main`
+    - ALWAYS branch off of the target receiving branch (i.e. `main` for the majority of work)
+    - This allows for peer review and approval
+2. All quality gates SHOULD be passed before review
+    - Linting
+    - Type-checking
+    - Isolation testing: unit, integration, functional
+    - Testing coverage thresholds
+3. All changes must be reviewed and approved by another contributor
+    - An author can not "approve" their own changes
+    - Some areas might have [code owners](../.github/CODEOWNERS.md) assigned requiring specific people to review
+    - Contributors should aim for a ratio of 1/1 (PRs reviewed/opened) or better (greater than 1)
+        - I've reviewed 3 and opened 5 would be 3/5 - less than 1
+        - I've reviewed 5 and opened 5 would be 5/5 - even at 1
+        - I've reviewed 5 and opened 3 would be 5/3 - greater than 1
```

Comment:
This probably isn't needed. Other internal teams probably won't be watching the repo or have time to. And it is unlikely external contributors will either.

Reply:
Removed.